README.md

```diff
@@ -17,15 +17,21 @@ npm i process-warning
 
 ### Usage
 
-The module exports a builder function that returns a utility for creating warnings and emitting them.
+The module exports two builder functions for creating warnings. A warning has a `emit` method.
 
 ```js
-const warning = require('process-warning')()
+const {
+  createWarning,
+  createDeprecation
+} = require('process-warning')
+
+const warning = createWarning('FastifyWarning', 'FSTWRN001', 'Hello %s', { unlimited: true })
```

Comment:
If we are redoing the whole interface, what is everyone's opinion of?:

```js
const warning = createWarning({
  name: 'ExampleWarning',
  code: 'EXMPWRN001',
  message: 'Hello %s',
  interpolationValues: ['world'],
  unlimited: true
})
```

Reply:
What is interpolationValues?

Reply:
Yeah, strike that one. That one should be used on the created warning.

Reply:
Done